airflow/providers/google/cloud/links/datafusion.py

```diff
@@ -30,8 +30,8 @@
 
 BASE_LINK = "https://console.cloud.google.com/data-fusion"
 DATAFUSION_INSTANCE_LINK = BASE_LINK + "/locations/{region}/instances/{instance_name}?project={project_id}"
-DATAFUSION_PIPELINES_LINK = "{uri}/cdap/ns/default/pipelines"
-DATAFUSION_PIPELINE_LINK = "{uri}/pipelines/ns/default/view/{pipeline_name}"
+DATAFUSION_PIPELINES_LINK = "{uri}/cdap/ns/{namespace}/pipelines"
+DATAFUSION_PIPELINE_LINK = "{uri}/pipelines/ns/{namespace}/view/{pipeline_name}"
```

Comment:
What is happen with already created links? Asked because current information which are stored into the XCom doesn't contain information about namespace

Reply:
@Taragolis Made some changes to support the backward compatibility of existing links. Existing links will continue behaving as is.